REVIEW.md

```diff
@@ -0,0 +1,85 @@
+Keep your history clean! There can be many commits or only one, as long as the story told is consistent.
+If you the history doesn't feel right, just be a negationist and use (and abuse) `git rebase -i`. 
+Just as long as you don't reright other people's history it's fine (really).
+
+And if you feel like, feel freer to `squash` your branch history into a single commit.
+
+## Pull/Merge request
+When ready you can open a Pull Request targeting the main branch.
+> BTW shall we rename `master` to `main`?
+
+OR not! You can open the PR as soon as you create your branch, 
+this can be relevant to keep other team member posted. 
+Just be sure to add the WIP keyword somewhere in the title and/or add the `WIP` label to your PR.
+
+> Also if your PR relate to an issue in any way, don't forget to mention it (#42 for issue number 42)
+> https://www.conventionalcommits.org/en/v1.0.0-beta.2/#commit-message-for-a-fix-using-an-optional-issue-number
+
+> __Ultimate tip__ If your branch only contains one commit with a title and a body:
+> ```
+> doc: Add review document draft
+> At last, here's the long-awaited documentation!
+> ```
+> The title and description of the PR will use this information.
+
+## Reviewer - *the peer*
+Depending on the subject of you can assign non technical people as reviewers:
+- member of the product to review doc, story or screeshot
+- member of the comm team to review spelling, relevance of keyword use, ...
+- member of UX UI or graphical, to review screenshot and user stories in practice, 
+> to do this the best way possible feature branches could be deployed and testable soemwhere though
+
+- At least 2 reviewer should be assigned to a PR.
+- Reviewers!
+  - Please be dilligent with your reviews! From our experience, PR which aren't reviewed soon tend to wait... wait.. sometime for months...even years! (I just closed a 7 years old PR recently)
+  - Tip: If you don't understand anything you're reading, you're might not be the right person for the job! And that's OK! 
+  Just let us know and we'll help you understand or choose another person fitting the task (at least try to look for typo:))
+  - We love our work to be read by others (who said egoless?!) so please try to show you care and leave some comments in the diff!
+- A PR should be merged in the next 72h? at most
+- You must have at least one approval to merge a PR
+
+### Unwritten rules (so far)
+Mind the state of the review :
+- approve
+- comment
+- request changes
+
+If the PR is approved, it's approved period, even if there are comments left, its just mean it's not these notices aren't that important after all.
+If one of the reviewer requested changes, changes must be applied or he must convinced otherwise until changes are approved. 
+And then request a re-review (there is a button for that)
+
+## Resources
+There are plenty of excellent resource out there feel freee to add anything relevant here
+
+## Tools
+Any tools that could help to be a git, gihub power user?
```

Comment:
```suggestion
If one of the reviewers requests changes, the changes must be applied or the reviewer must be convinced otherwise until the PR is approved.
Once changes are applied, the PR author should then request a re-review (there is a button for that).

## Resources
There are plenty of excellent resources out there, so feel free to add anything relevant here.

## Tools
Any tools that could help folks to become a Git, Github power user?

- [Managing your work on GitHub](https://docs.github.com/en/github/managing-your-work-on-github)
- [Using Git](https://docs.github.com/en/github/using-git)
- [A Step-by-Step Guide to Git](https://opensource.com/article/18/1/step-step-guide-git)
- [How to become a Git expert](https://www.freecodecamp.org/news/how-to-become-a-git-expert-e7c38bf54826/)
```